Dataflow does not run with Java 11

Attempting to run a generated Dataflow app on Java 11 fails:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.beam.repackaged.beam_runners_direct_java.com.google.protobuf.UnsafeUtil (file:/C:/Users/IEUser/.m2/repository/org/apache/beam/beam-runners-direct-java/2.5.0/beam-runners-direct-java-2.5.0.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of org.apache.beam.repackaged.beam_runners_direct_java.com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Exception in thread "main" org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.util.concurrent.UncheckedExecutionException: java.lang.UnsupportedOperationException: Cannot define class using reflection: Could not find sun.misc.Unsafe
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2214)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache.get(LocalCache.java:4053)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:4057)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4986)
	at org.apache.beam.runners.direct.DoFnLifecycleManager.get(DoFnLifecycleManager.java:61)
	at org.apache.beam.runners.direct.ParDoEvaluatorFactory.createEvaluator(ParDoEvaluatorFactory.java:124)
	at org.apache.beam.runners.direct.ParDoEvaluatorFactory.forApplication(ParDoEvaluatorFactory.java:78)
	at org.apache.beam.runners.direct.TransformEvaluatorRegistry.forApplication(TransformEvaluatorRegistry.java:178)
	at org.apache.beam.runners.direct.DirectTransformExecutor.run(DirectTransformExecutor.java:118)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.UnsupportedOperationException: Cannot define class using reflection: Could not find sun.misc.Unsafe
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection$Dispatcher$Initializable$Unavailable.defineClass(ClassInjector.java:334)
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.loading.ClassInjector$UsingReflection.inject(ClassInjector.java:187)
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.loading.ClassLoadingStrategy$Default$InjectionDispatcher.load(ClassLoadingStrategy.java:205)
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.loading.ClassLoadingStrategy$Default.load(ClassLoadingStrategy.java:126)
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.TypeResolutionStrategy$Passive.initialize(TypeResolutionStrategy.java:79)
	at org.apache.beam.repackaged.beam_sdks_java_core.net.bytebuddy.dynamic.DynamicType$Default$Unloaded.load(DynamicType.java:4247)
	at org.apache.beam.sdk.transforms.reflect.ByteBuddyDoFnInvokerFactory.generateInvokerClass(ByteBuddyDoFnInvokerFactory.java:340)
	at org.apache.beam.sdk.transforms.reflect.ByteBuddyDoFnInvokerFactory.getByteBuddyInvokerConstructor(ByteBuddyDoFnInvokerFactory.java:240)
	at org.apache.beam.sdk.transforms.reflect.ByteBuddyDoFnInvokerFactory.newByteBuddyInvoker(ByteBuddyDoFnInvokerFactory.java:213)
	at org.apache.beam.sdk.transforms.reflect.ByteBuddyDoFnInvokerFactory.newByteBuddyInvoker(ByteBuddyDoFnInvokerFactory.java:144)
	at org.apache.beam.sdk.transforms.reflect.DoFnInvokers.invokerFor(DoFnInvokers.java:35)
	at org.apache.beam.runners.direct.DoFnLifecycleManager$DeserializingCacheLoader.load(DoFnLifecycleManager.java:102)
	at org.apache.beam.runners.direct.DoFnLifecycleManager$DeserializingCacheLoader.load(DoFnLifecycleManager.java:91)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3628)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2336)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2295)
	at org.apache.beam.repackaged.beam_runners_direct_java.com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2208)
	... 13 more
```

--- a/plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowProjectCreator.java
+++ b/plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/DataflowProjectCreator.java
@@ -29,9 +29,7 @@ import com.google.common.base.Strings;
 import java.io.File;
 import java.lang.reflect.InvocationTargetException;
 import java.net.URI;
-import java.util.Arrays;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.EnumSet;
 import java.util.List;
 import java.util.Properties;
@@ -56,12 +54,6 @@ import org.eclipse.m2e.core.project.ProjectImportConfiguration;
  * An {@code IRunnableWithProgress} that creates a new Cloud Dataflow Java Project.
  */
 public class DataflowProjectCreator implements IRunnableWithProgress {
-
-  private static final String DEFAULT_JAVA_VERSION = JavaCore.VERSION_1_8;
-  private static final List<String> JAVA_VERSION_BLACKLIST =
-      Collections.unmodifiableList(Arrays.asList(JavaCore.VERSION_1_1, JavaCore.VERSION_1_2,
-          JavaCore.VERSION_1_3, JavaCore.VERSION_1_4, JavaCore.VERSION_1_5, 
-          JavaCore.VERSION_1_6, JavaCore.VERSION_1_7, JavaCore.VERSION_CLDC_1_1));
 
   private final IProjectConfigurationManager projectConfigurationManager;
 
@@ -170,7 +162,8 @@ public class DataflowProjectCreator implements IRunnableWithProgress {
     archetype.setArtifactId(template.getArtifactId());
 
     Properties archetypeProperties = new Properties();
-    archetypeProperties.setProperty("targetPlatform", getTargetPlatform());
+    // only support 1.8 until we support Beam
+    archetypeProperties.setProperty("targetPlatform", JavaCore.VERSION_1_8);
 
     IPath location = null;
     if (customLocation) {
@@ -224,32 +217,6 @@ public class DataflowProjectCreator implements IRunnableWithProgress {
     prefs.save();
   }
 
-  /**
-   * Gets the target platform of the environment's JDT plugin. If not found, use the default target
-   * platform. If not supported, throw a {@link ProjectCreationException}.
-   */
-  private static String getTargetPlatform() throws ProjectCreationException {
-    String targetPlatform = JavaCore.getOption(JavaCore.COMPILER_CODEGEN_TARGET_PLATFORM);
-    if (targetPlatform == null || JAVA_VERSION_BLACKLIST.contains(targetPlatform)) {
-      DataflowCorePlugin.logWarning(
-          "Couldn't find supported target platform. Got %s. Using default version %s",
-          targetPlatform, DEFAULT_JAVA_VERSION);
-      targetPlatform = DEFAULT_JAVA_VERSION;
-    } else {
-      targetPlatform = targetPlatform.toString();
-    }
-    return targetPlatform;
-  }
-
-  private DataflowProjectValidationStatus validateTargetPlatform() {
-    try {
-      getTargetPlatform();
-      return DataflowProjectValidationStatus.OK;
-    } catch (ProjectCreationException e) {
-      return DataflowProjectValidationStatus.UNSUPPORTED_TARGET_PLATFORM;
-    }
-  }
-
   private void checkCancelled(IProgressMonitor monitor) {
     if (monitor.isCanceled()) {
       throw new OperationCanceledException();
@@ -267,7 +234,6 @@ public class DataflowProjectCreator implements IRunnableWithProgress {
     statuses.add(validateMavenGroupId());
     statuses.add(validateMavenArtifactId());
     statuses.add(validatePackage());
-    statuses.add(validateTargetPlatform());
 
     // Must be last: Remove OK so the set contains only failed validations.
     statuses.remove(DataflowProjectValidationStatus.OK);

